packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.routing.js

```diff
@@ -198,6 +199,12 @@ export default /* @ngInject */ ($stateProvider) => {
             ),
       serviceMonitoring: /* @ngInject */ ($stateParams, Server) =>
         Server.getAllServiceMonitoring($stateParams.productId),
+      technicalDetails: /* @ngInject */ ($http, $stateParams) =>
+        $http
+          .get(`/dedicated/technical-details/${$stateParams.productId}`, {
```

Comment:
`$stateParams` are now deprecated (https://ui-router.github.io/ng1/docs/latest/modules/injectables.html#_stateparams), using `$transition` would be better 
Also can't we inject serverName instead ? https://github.com/ovh/manager/blob/7aff9155edd0f312097f028db95e9a22c28146aa/packages/manager/apps/dedicated/client/app/dedicated/server/server.routing.js#L28